extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -252,20 +329,15 @@ export class AmpAutocomplete extends AMP.BaseElement {
       : null;
     this.submitOnEnter_ = this.element.hasAttribute('submit-on-enter');
     if (this.element.hasAttribute('suggest-first')) {
-      if (this.filter_ === FilterType.PREFIX) {
-        this.suggestFirst_ = true;
-      } else {
-        user().error(
-          TAG,
-          '"suggest-first" requires "filter" to be prefix.' +
-            ' Unexpected "filter" type: ' +
-            this.filter_
-        );
-      }
+      this.suggestFirst_ = this.filter_ === FilterType.PREFIX || this.inline_;
+      userAssert(
+        this.suggestFirst_,
+        '"suggest-first" requires "filter" to equal "prefix". ' +
+          'If "inline" is present, "filter" can be any value.' +
+          ' Unexpected "filter" type: ' +
+          this.filter_
```

Comment:
Associate element here too. 

Super nit, these three string lines may fit in one, and the last string should probably be interpolated for conciseness.